keystorage/aws/src/test/java/tech/pegasys/signers/aws/AwsSecretsManagerTest.java

```diff
@@ -265,9 +263,34 @@ private void closeClients() {
     closeTestSecretsManager();
   }
 
-  private String createSecret(final Tag tag, String secretValue)
+  private boolean checkTestSecretExists(
+      final String testSecretName, final Tag tag, final String secretValue) {
+    try {
+      final GetSecretValueResponse getSecretValueResponse =
+          testSecretsManagerClient
+              .getSecretValue(GetSecretValueRequest.builder().secretId(testSecretName).build())
+              .get(30, TimeUnit.SECONDS);
+      final DescribeSecretResponse describeSecretResponse =
+          testSecretsManagerClient
+              .describeSecret(DescribeSecretRequest.builder().secretId(testSecretName).build())
+              .get(30, TimeUnit.SECONDS);
+      return getSecretValueResponse.name().equals(testSecretName)
```

Comment:
If only the tag or value changed will create secret change that? or will it fail creating the secrets before tag changes are applied?